<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabledetailwidget</class>
 <widget class="QWidget" name="tabledetailwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>参数配置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="configTableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>点表数据信息</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>类型过滤：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="pointTypeCombox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnAdd">
           <property name="text">
            <string>新增</string>
           </property>
           <property name="icon">
            <iconset resource="../resource/sysconfig.qrc">
             <normaloff>:/add.png</normaloff>:/add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelete">
           <property name="text">
            <string>删除</string>
           </property>
           <property name="icon">
            <iconset resource="../resource/sysconfig.qrc">
             <normaloff>:/delete.png</normaloff>:/delete.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnExport">
           <property name="text">
            <string>导出</string>
           </property>
           <property name="icon">
            <iconset resource="../resource/sysconfig.qrc">
             <normaloff>:/excel.png</normaloff>:/excel.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCommitChanges">
           <property name="text">
            <string>保存</string>
           </property>
           <property name="icon">
            <iconset resource="../resource/sysconfig.qrc">
             <normaloff>:/save.png</normaloff>:/save.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRollback">
           <property name="text">
            <string>回滚</string>
           </property>
           <property name="icon">
            <iconset resource="../resource/sysconfig.qrc">
             <normaloff>:/download.png</normaloff>:/download.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="pointTableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/sysconfig.qrc"/>
 </resources>
 <connections/>
</ui>
